src/omnidirectional_controller.cpp

```diff
@@ -97,11 +97,16 @@ InterfaceConfiguration OmnidirectionalController::command_interface_configuratio
 }
 
 InterfaceConfiguration OmnidirectionalController::state_interface_configuration() const {
-  std::vector<std::string> conf_names;
-  for (const auto & joint_name : wheel_names_) {
-    conf_names.push_back(joint_name + "/" + HW_IF_VELOCITY);
+  if (odom_params_.open_loop) { //return an empty configuration if "open loop" mode is activated.
+    return {interface_configuration_type::NONE, {}};
+  } 
+  else {
```

Comment:
This else clause here is redundant because there is the `return` statement inside the `if`scope. Thus, if the `if` is satisfied, the method return already in line 101

Reply:
You're right, it's an unnecessary redundancy. My bad coding habits die hard. I will change this.

Reply:
The else clause returned?